.github/release-drafter.yml

```diff
@@ -0,0 +1,31 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
```

Comment:
with all the labels in place, our PRs should be labelled accordingly

Reply:
As a contributor, I do not have permissions to assign labels to my PR currently

Reply:
@reetasingh you should have permissions to add labels now